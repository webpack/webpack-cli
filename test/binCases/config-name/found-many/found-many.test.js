"use strict";

const { run } = require("../../../testUtils");

test("found-many", () => {
	const { code, stdout, stderr } = run(__dirname, [
		"--config",
		"./webpack.config.js",
		"--config-name",
		"bar",
		"--output-filename",
		"[name].js",
		"--output-chunk-filename",
		"[id].chunk.js",
		"--target",
		"async-node",
		"--mode",
		"production"
	]);

	expect(code).toBe(0);
	expect(stdout).toEqual(expect.anything());
	expect(stdout).toContain("./index2.js");
	expect(stdout).toContain("./index3.js");
	expect(stderr).toHaveLength(0);
	expect(stdout).toMatchSnapshot();
});
