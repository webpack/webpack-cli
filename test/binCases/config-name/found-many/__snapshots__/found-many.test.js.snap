// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`found-many 1`] = `
"Child bar:
      Asset       Size  Chunks             Chunk Names
    main.js  945 bytes       0  [emitted]  main
    Entrypoint main = main.js
    [0] ./index2.js 24 bytes {0} [built]
Child bar:
      Asset       Size  Chunks             Chunk Names
    main.js  930 bytes       0  [emitted]  main
    Entrypoint main = main.js
    [0] ./index3.js 0 bytes {0} [built]"
`;
